apps/andi/lib/andi_web/live/dataset_live_view.ex

```diff
@@ -0,0 +1,92 @@
+defmodule AndiWeb.DatasetLiveView do
+  use Phoenix.LiveView
+  alias AndiWeb.Router.Helpers, as: Routes
+
+  def render(assigns) do
+    ~L"""
+    <div class="datasets-index">
+      <h1 class="datasets-index__title">All Datasets</h1>
+      <div class="datasets-index__search">
+        <form phx-change="search">
+          <label for="datasets-index__search">
+          Search:
+          <input
+            name="search-value"
+            phx-debounce="250"
+            class="datasets-search"
+            type="text"
+            id="datasets-index__search"
+            value="<%= @search_text %>"
+          >
+          </label>
+        </form>
+      </div>
+      <div class="datasets-index__table">
+        <table class="datasets-table">
+          <thead>
+            <th class="datasets-table__th datasets-table__cell">Organization</th>
+            <th class="datasets-table__th datasets-table__cell">Dataset Name</th>
+          </thead>
+          <%= case @datasets do %>
+            <% nil -> %>
+            <tr><td class="datasets-table__cell" colspan="100%">Loading datasets...</td></tr>
+            <%= [] -> %>
+              <tr><td class="datasets-table__cell" colspan="100%">No Datasets Found</td></tr>
+            <% _ -> %>
+            <%= for dataset <- @datasets do %>
+              <tr class="datasets-table__tr">
+                <td class="datasets-table__cell datasets-table__cell--break"><%= dataset.business.orgTitle %></td>
+                <td class="datasets-table__cell datasets-table__cell--break"><%= dataset.business.dataTitle %></td>
+              </tr>
+            <% end %>
+          <% end %>
+        </table>
+      </div>
+    </div>
+    """
+  end
+
+  def mount(_session, socket) do
+    {:ok, assign(socket, datasets: nil, search_text: "")}
+  end
+
+  def handle_params(params, _uri, socket) do
+    search_text = Map.get(params, "search-value", "")
+    socket_pid = self()
+
+    Task.start(fn ->
+      # Process.sleep(1000)
+      datasets = Andi.Services.DatasetRetrieval.get_all!()
```

Comment:
Ultimately, I would rather we keep a list of all datasets in the state and update it through a PubSub. But I don't know that we need to go that far for this card. Honestly, I don't even know that I'd worry about blocking the render atm.